scripts/release/build-packages.sh

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env bash
```

Comment:
this file was used for when we cowboyed building the on prem artifacts by replaying and editing the old ksqldb-release jenkins job. Once we finish repurposing the old ksqldb-release jenkins job it will continue to use this file to build the on prem artifacts